src/Mono.Android/Android.App/SyncContext.cs

```diff
@@ -14,7 +15,15 @@ public override SynchronizationContext CreateCopy ()
 
 		public override void Post (SendOrPostCallback d, object state)
 		{
-			using (Handler h = new Handler(Application.Context.MainLooper)) {
+			var looper = Application.Context?.MainLooper;
+			if (looper == null) {
+				var message = "No Looper available to this SynchronizationContext, callback will not be executed.";
+				if (JNIEnv.IsRunningOnDesktop)
+					message += " Using await when running on the Android Designer is not currently supported.";
+				Logger.Log (LogLevel.Error, "monodroid-synchronizationcontext", message);
```

Comment:
Logger is good we get it, tag doesn't matter as long as it stays consistent that's what I can catch on the IDE side.